modules/virt-troubleshooting-vm-import.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+// * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
+[id='virt-troubleshooting-vm-import_{context}']
+= Troubleshooting a virtual machine import
+
+[id='logs_{context}']
+== Logs
+
+ifdef::virt-importing-vmware-vm[]
+You can check the V2V Conversion Pod log for errors.
+
+.Procedure
+
+. Obtain the V2V Conversion Pod name:
++
+----
+$ oc get pods -n <project> | grep v2v <1>
+kubevirt-v2v-conversion-f66f7d-zqkz7            1/1     Running     0          4h49m
+----
+<1> Specify the project of your imported virtual machine.
+
+. Obtain the V2V Conversion Pod log:
++
+----
+$ oc logs kubevirt-v2v-conversion-f66f7d-zqkz7 -f -n <project>
+----
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+You can check the VM Import Controller Pod log for errors.
+
+.Procedure
+
+. Obtain the VM Import Controller Pod name:
++
+----
+$ oc get pods -n <project> | grep import <1>
+vm-import-controller-f66f7d-zqkz7            1/1     Running     0          4h49m
+----
+<1> Specify the project of your imported virtual machine.
+
+. Obtain the VM Import Controller Pod log:
++
+----
+$ oc logs vm-import-controller-f66f7d-zqkz7 -f -n <project>
+----
+endif::[]
+
+[id='error-messages_{context}']
+== Error messages
+
+The following error messages might appear:
+
+ifdef::virt-importing-rhv-vm[]
+* The following error message is displayed in the VM Import Controller Pod log if the RHV VM's system settings do not emulate the Intel Q35 chipset:
++
+----
+The virtual machine could not be imported.
+MappingRulesVerificationFailed: VM uses unsupported bios type: i440fx_sea_bios
+----
+
+* The following error message is displayed in the VM Import Controller Pod log if the target VM name exceeds 23 characters link:https://bugzilla.redhat.com/show_bug.cgi?id=1857165[(*BZ#1857165*)]:
++
+----
+Message:               Error while importing disk image
+Reason:                ProcessingFailed
+----
+
+* The following error message is displayed in the VM Import Controller Pod log and the progress bar stops at 10% if the {VirtProductName} storage PV is not suitable:
++
+----
+Failed to bind volumes: provisioning failed for PVC
+----
++
+You must use Ceph Filesystem or local storage. Cinder storage is not supported. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857784[(*BZ#1857784*)]
```

Comment:
You must use Ceph Filesystem or local storage. Cinder storage
Should be:
You must use Ceph Filesystem or NFS. Standard Cinder storage

Reply:
@ilstein1 
This is now NFS only, right? For RHV VM